java-agent/src/nativeMain/kotlin/com/epam/drill/agent/Agent.kt

```diff
@@ -87,11 +88,18 @@ object Agent {
 
     private val pathSeparator = if (Platform.osFamily == OsFamily.WINDOWS) "\\" else "/"
 
+    private fun getDrillInstallationDir() = open("/proc/self/cmdline", O_RDONLY).let(::Input).readText().run {
```

Comment:
`"/proc/self/cmdline"`- is this crossplatform?

I have no idea if prev implementation ([JavaProcessInfo](https://github.com/Drill4J/java-agent/blob/main/java-agent/src/nativeMain/kotlin/com/epam/drill/agent/configuration/process/JavaProcessInfo.kt#L34C43-L34C43) and ["ps -p *pid* -o args"](https://github.com/Drill4J/lib-jvm-shared/blob/main/agent/src/posixMain/kotlin/com/epam/drill/agent/configuration/FileUtils.kt#L25) ) were crossplatform either

Reply:
Refactored to be crossplatform.